packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -0,0 +1,256 @@
+import {Schedule} from '@aws-cdk/aws-events';
+import {Construct, Duration, IResource, Lazy, Resource} from '@aws-cdk/core';
+import {CfnBackupPlan} from './backup.generated';
+import {IBackupVault} from './vault';
+
+/**
+ * Represents the Lifecycle configuration of the backup resources.
+ */
+export interface Lifecycle {
+  /**
+   * Specifies how long a backup lasts before being deleted.
+   * Once a backup is moved into a cold storage class, it has to live 90 days
+   * before it can be deleted, therefore this value has to be 90 days greater.
+   */
+  readonly deleteAfter: Duration;
+  /**
+   * Specifies how long a backup will stay in warm storage before being moved
+   * to a cold storage class.
+   */
+  readonly moveToColdStorageAfter: Duration;
+}
+
+/**
+ * Represents the backup plan rule configuration.
+ * If a vault is provided, this will be used instead of the one from the backup plan.
+ */
+export interface PlanRule {
```

Comment:
I would make a `class` of it. This allows to offer nice static methods like `public static daily(backupVault: IBackupVault)`, `public static weekly(backupVault: IBackupVault)`, `public static monthly1Year(backupVault: IBackupVault)`...

Reply:
Just wondering what hour of the day would be the best default to execute the backups?